open-api-spec.yaml

```diff
@@ -285,10 +287,12 @@ paths:
             schema:
               type: object
               properties:
-                authorized:
-                  type: boolean
-                  description: Indicates if the payment is authorized to begin processing.
-                  default: false
+                quoteId:
```

Comment:
Should an outgoing payment for a quote be created via the quote url instead?
```
POST https://openpayments.guide/alice/quotes/ab03296b-0c8b-4776-b94e-7ee27d868d4d/outgoing-payment
```

Reply:
I think that is messy. I think an outgoing payment should always be created via the open payments url and `quote` is another field in the body.